.github/workflows/test-mobile-codecheck.yml

```diff
@@ -0,0 +1,42 @@
+name: "[Test] Mobile App Codecheck"
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - release
+      - hotfix
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
i think this will pose a problem in the case of a workflow_dispatch as github.ref will probably by undefined or will use `develop` which is not what we want ? Also it seems `workflow_dispatch` triggers does not take any type of `ref` as input so we only checkout on `develop` ?

Reply:
I thought the solution for now was just to run the cron job on `develop`. I think for now since it won't be running on feature branches we can probably just remove the concurrency, and add it back in. I'll look into the ref (since I think it's still useful to be able to run the tests against certain branches)